server/tserver/src/main/java/org/apache/accumulo/tserver/memory/LargestFirstMemoryManager.java

```diff
@@ -145,6 +147,12 @@ protected long getMinCIdleThreshold(KeyExtent extent) {
     return mincIdleThresholds.get(tableId);
   }
 
+  protected long getMaxAge(KeyExtent extent) {
+    TableId tableId = extent.tableId();
+    return mincAgeThresholds.computeIfAbsent(tableId, tid -> context.getTableConfiguration(tid)
```

Comment:
Does this make this static, it won't respond to a runtime change in the property value? If so, could use a Caffeine cache here with expiration.

Reply:
Yeah it does make it static, I'll switch it to use a cache.

Reply:
Also a cache would  eventually clean out deleted tables from the map.